wearApp/src/main/java/dev/johnoreilly/confetti/wear/MainActivity.kt

```diff
@@ -24,7 +24,7 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        val startingConferenceAsync = lifecycleScope.async {
+        lifecycleScope.async {
```

Comment:
I guess we can remove this now? I can do it later.

Reply:
It was triggering a warning, feel free to remove 👍